lib/l10n/l10n.dart

```diff
@@ -127,6 +128,52 @@ class Language {
   String toString() => locale.toLanguageTag();
 }
 
+/// Returns bytes formatted into a human-readable, localized string.
+///
+/// ```dart
+/// print(formattedFileSize(null)); // ...
+/// print(formattedFileSize(0)); // 0 B
+/// print(formattedFileSize(10)); // 10 B
+/// print(formattedFileSize(1024)); // 1 KB
+/// print(formattedFileSize(1024 + 50)); // 1 KB
+/// print(formattedFileSize(1024 + 100)); // 1.1 KB
+/// ```
+String formattedFileSize(int? bytes) {
+  String valueToString(double value) {
+    final string = value.toStringAsFixed(1);
+
+    return string.endsWith('.0')
+        ? string.substring(0, string.length - 2)
+        : string;
```

Comment:
Oh, I see now what you meant there. Well, I don't really see a problem printing `2.0 MB` instead of `2 MB`, for example. Otherwise I suppose it may confuse the user: imaging seeing `2 MB` and wondering whether it is actually `2.0` or perhaps a `1.9 MB` or `2.1 MB` and the application just rounds the value? Being concrete there won't hurt? What're your thoughts?

Reply:
Ok, I'll remove the trimming of trailing ".0"